armi/reactor/components/volumetricShapes.py

```diff
@@ -168,6 +168,10 @@ class RadialSegment(ShapedComponent):
         |-----------------------> radius, X
         |
         |
+
+    .. impl:: Compute properties of radial segment
+        :id: I_ARMI_COMP_FLUID0
+        :implements: R_ARMI_COMP_FLUID
```

Comment:
Delete this one.

I'm not sure WHERE this will go, probably in a few places, as the logic is complicated.

But not here, anyway.